src/IdentityRegistry.ts

```diff
@@ -18,12 +21,9 @@ export type Identity = string | symbol;
  * A registry of values, mapped by identities.
  */
 export default class IdentityRegistry<V extends Object> {
-	private _entryMap: Map<Identity, Entry<V>>;
-	private _idMap: WeakMap<V, Identity>;
-
 	constructor() {
-		this._entryMap = new Map<Identity, Entry<V>>();
-		this._idMap = new WeakMap<V, Identity>();
+		entryMaps.set(this, new Map<Identity, Entry<V>>());
```

Comment:
Sorry, to make you do this again, but we have identified an ideal pattern... When we have a 1:1 relationship between private entries in a WeakMap, we should create an interface for managing them and store only one value.  See: dojo/meta#74.  (Or feel free to input on why it is a stupid pattern and we should do something else)